makelib/golang.mk

```diff
@@ -360,21 +318,15 @@ $(GOFMT):
 
 $(GOIMPORTS):
 	@$(INFO) installing goimports
-	@mkdir -p $(TOOLS_HOST_DIR)/tmp-goimports || $(FAIL)
-	@GO111MODULE=off GOPATH=$(TOOLS_HOST_DIR)/tmp-goimports GOBIN=$(TOOLS_HOST_DIR) $(GOHOST) get golang.org/x/tools/cmd/goimports || rm -fr $(TOOLS_HOST_DIR)/tmp-goimports || $(FAIL)
-	@rm -fr $(TOOLS_HOST_DIR)/tmp-goimports
+	@GOBIN=$(TOOLS_HOST_DIR) $(GOHOST) install golang.org/x/tools/cmd/goimports@latest || $(FAIL)
```

Comment:
Could we pin this version that's being pulled so that we don't get surprises between local and CI?